rust-toolchain.toml

```diff
@@ -0,0 +1,2 @@
+[toolchain]
+channel = "stable"
```

Comment:
Sneaking this in again through the backdoor huh?
:rofl: jk, keep it in.

Reply:
oh bruh, I forgot it. Sorry! :rofl: 

> rofl jk, keep it in.

oh lol, if you insist, I'd like to :)